### ISSUE TITLE
Jade/Pug comment syntax

### ISSUE THREAD

**Issue by cg-cnu** [Microsoft/vscode#9065]
- VSCode Version: 1.3.0
- OS Version: CentOS Linux release 7.1.1503 (Core)

Jade/Pug comment syntax highlighting is behaving odd!
Am not exactly sure if that's the issue with VScode or the way I am writing the comment syntax.
[Video](https://youtu.be/4S0GmSLFdh8)

**Comment by weinand** [comment]
@cg-cnu please provide more detailed steps what you are typing and how what you are seeing is different from what you are expecting.

**Comment by cg-cnu** [comment]
From the video...
* When I added a new line above the comment It worked for the first time(syntax highlighting is proper in all the places).
* Next time only the line in which I added the new line didn't have the proper syntax highlighting(becomes white instead of green).
* At last, when I added the new line, syntax highlighting for the whole code below the new line is broken(random colors ).

Basically, if I add a new line above the comment line the syntax highlighting is broken. But, its a bit random as shown in the video. Let me know if that helps or point me, if am doing anything wrong.
